core/ui/public/i18n/en/translation.json

```diff
@@ -608,14 +608,14 @@
         "http_route_details": "HTTP route details",
         "url": "URL",
         "strip_prefix": "Strip URL path prefix",
-        "strip_prefix_tooltip": "E.g.: <code>http://{host}/{path}/{requestpath}?{querystring}</code> is mapped to <code>http://{host}/{requestpath}?{querystring}</code>",
+        "strip_prefix_tooltip": "The prefixes option defines the prefixes to strip from the request URL. E.g.: It allows you to strip a specified prefix from the URL path. If, for example, you remove /api from /api/v1/resource, the resulting path after it is applied would be /v1/resource.",
```

Comment:
This is not very clear to me: what and where is "prefixes option"?

Reply:
associate to https://doc.traefik.io/traefik/middlewares/http/stripprefix/

Reply:
I assume the previous explanation was not much clear hence davidep wanted to enhance it

Reply:
Then I'd call it "Traefik prefixes option" and add a link to https://doc.traefik.io/traefik/middlewares/http/stripprefix/